src/bundles/ApplicationsAdmin/components/RevertNotification/RevertNotification.js

```diff
@@ -1,92 +1,114 @@
 
-import Layout        from '../../../../shared/Layout';
-import BaseForm      from '../../../../shared/form/BaseForm';
-import Textarea     from '../../../../shared/form/Textarea';
-import formProps     from '../../../../shared/reduxModules/formPropsSelector';
 
-import './RevertNotification.css'
-
-class RevertNotificationForm extends BaseForm {
+export default class RevertedNotificationForm extends Component {
 
   static propTypes = {
-    action: React.PropTypes.string,
-    csrf_token: React.PropTypes.string,
-    form: React.PropTypes.object.isRequired,
-    action: React.PropTypes.string,
-    model: React.PropTypes.oneOfType([
-      React.PropTypes.string,
-      React.PropTypes.func
-    ]).isRequired,
-    id: React.PropTypes.number,
-    onSubmit: React.PropTypes.func.isRequired,
-    altSubmit: React.PropTypes.func
+    appID: React.PropTypes.number.isRequired,
+    onClose: React.PropTypes.func.isRequired,
+    message: React.PropTypes.string.isRequired
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      message: templateString
+    };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(e) {
+    this.setState({
+      message: e.target.value
+    })
+  }
+
+  resetMessageText(propsMessage) {
+    this.setState({
+      message: propsMessage
+    });
+  }
+
   render() {
     const {
-      action,
-      csrf_token,
-      model,
-      revertEmailForm,
-      onSubmitFailed,
-      onSubmit,
-      altSubmit,
-      id
+      message,
+      appID,
+      onClose,
+      onRevertClick
     } = this.props;
 
     return (
-      <Layout>
-        <header>
-          <h2>Application reversion email message</h2>
-        </header>
-        <article role="main">
-
-          <Form
-            model={model}
-            action={action}
-            method="post"
-            id="revertNotification"
-            onSubmitFailed={onSubmitFailed}
-          >
-
-            <input type="hidden" name="csrf_token" id="csrf_token" value={csrf_token}/>
-
-            <Textarea
-              model={`${model}.message`}
-              name="message"
-              id="message"
-              htmlFor="message"
-              label="Email Message"
-            />
-            <div styleName="form-button-wrapper">
-              <button style={{ width: '100px' }} onClick={() => {
-                onSubmit(id, revertEmailForm.message);
-                altSubmit(id);
-              }}>Send
-              </button>
-            </div>
-          </Form>
-        </article>
-      </Layout>
+      <form>
+        <label htmlFor="formHeading">
+          <h3>
+            Revert application with optional email to seller
+          </h3>
+          <b>
+            {`Provide a short clear reason, starting with a hyphen. Eg "${"- ABN missing a digit"}"`}
+          </b>
+          <textarea value={this.state.message} onChange={this.handleChange}/>
```

Comment:
doesn't seem quite right the way this is setup - why put textare value in state rather than submit form? Hang on are you following https://facebook.github.io/react/docs/forms.html?